framework/src/modules/interoperability/mainchain/commands/submit_mainchain_cross_chain_update.ts

```diff
@@ -84,7 +100,19 @@ export class SubmitMainchainCrossChainUpdateCommand extends BaseCrossChainUpdate
 			this._verifyLivenessConditionForRegisteredChains(context);
 		}
 
-		await this.verifyCommon(context);
```

Comment:
We can remove this function as its not common anymore and only called on the sidechain CCU update. Created relevant refactoring issue ->  https://github.com/LiskHQ/lisk-sdk/issues/8258